kin-sdk/kin-sdk-lib/src/main/java/kin/sdk/exception/TransactionFailedException.java

```diff
@@ -15,16 +15,16 @@
 
     public TransactionFailedException(@Nullable String txResultCode,
         @Nullable List<String> opResultCode) {
-        super(getMessage(opResultCode));
+        super(getMessage(txResultCode, opResultCode));
 
         this.txResultCode = txResultCode;
         this.opResultCode = opResultCode;
     }
 
     @NonNull
-    private static String getMessage(@Nullable List<String> opResultCode) {
+    private static String getMessage(@Nullable String txResultCode, @Nullable List<String> opResultCode) {
         return opResultCode != null && !opResultCode.isEmpty() ?
-            "Transaction failed with the error = " + opResultCode.get(0) :
+            "Transaction failed with the error = " + txResultCode + ", operations = " + opResultCode :
```

Comment:
adding `txResultCode` anyway is great, just had a problem where `opResultCode` was null while `txResultCode` is not, and I wasn't able to understand the problem from the logs. (it was `tx_bad_seq`)
but, you should print it even when `opResultCode` is null/empty but `txResultCode` is not, o.w. in the aforementioned case we'll get nothing.

Reply:
It was like that before I added the changed.
Anyway I just added it.